### PR TITLE
Minor improvements to data tracing

### PR DESCRIPTION
**Pull request by sglow**
Timestamps are added to saved trace data
controller is queried to find sample time rather then hard code it

### CODE REVIEW

**Comment by sglow** [comment]
Which PR was that?  The original one for the debug interface from several weeks ago?

**Comment by sglow** [comment]
I looked back through this PR https://github.com/RespiraWorks/VentilatorSoftware/pull/302 and didn't really see anything outstanding related to the python code.  Is there something in particular you're thinking of?

**Comment by jlebar** [comment]
I was thinking of https://github.com/RespiraWorks/VentilatorSoftware/pull/458/files#diff-b279f8268b36cec1013b705589b0b92e.

None of it is major.

**Comment by sglow** [comment]
Oh, right.  No problem I'll go through that.

File: utils/controller_debug.py [hunk 1]
```diff
@@ -699,12 +704,13 @@ def TraceSaveDat(dat, fname, separator="  "):
...
 
     fp = open(fname, "w")
-    line = ["time"]
+    line = ["time(sec)"]
     for v in tv:
         line.append(v.name)
     fp.write(separator.join(line) + "\n")
 
     for i in range(len(dat[0])):
+        # First colume is time in seconds
```
Comment (jlebar) [inline utils/controller_debug.py:713]: colume

Comment (sglow) [reply utils/controller_debug.py:713]: Doh, that's why I keep misspelling volume as volumn.  English is a weird language

**Pull request merged**